### PR TITLE
Alerts v3

### PR DESCRIPTION
**Pull request by Roshannair24**
updated alerts
alerrt_v3

### CODE REVIEW

**Review by VedanthB — APPROVED** [review]
Good job Roshan,
Looks good to go 🚀

**Review by amarphule — CHANGES_REQUESTED** [review]
please correct why there are two alerts same names with different color
alt

File: components.html [hunk 1]
```diff
@@ -1781,10 +1765,7 @@ <h2>Toast</h2>
               </div>
             </div>
 
-            <div
-              class="toast-iframe"
-              style="background-color: rgb(255, 217, 0)"
```
Comment (Kavita539) [inline components.html:1786]: Suggestion : you can use variables for defining colors

**Review by Kavita539 — APPROVED** [review]
Overall looks good

**Pull request merged**